src/discussions/posts/post/PostLink.jsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 
-import { faCircle, faFlag } from '@fortawesome/free-solid-svg-icons';
+import { faCircle } from '@fortawesome/free-solid-svg-icons';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
```

Comment:
is now available in paragon with name Unread

Reply:
Updated to use this new icon, thanks!